proto/conduit/v1/schema.proto

```diff
@@ -35,7 +35,8 @@ message CreateSchemaResponse {
 }
 
 message GetSchemaRequest {
-  string id = 1;
+  string name = 1;
```

Comment:
At some point I heard you mention that we could rename `name` to `subject`, is this the plan here?

Reply:
I was thinking about that, but I'm leaning more towards keeping `name`. It's a bit more generic and given that we plan to move to our own schema registry.

Reply:
When we say "our own" it will still need to be compatible with the Confluent schema registry, otherwise we would be losing compatibility with other tools like Kafka Connect that only know how to talk to the Confluent schema registry.

I personally also prefer `name`, it's straight forward, don't get me wrong, but it might create unnecessary friction, since it's essentially the same as `subject` in the Confluent world.

Reply:
I don't have much against `subject` and I don't think we're choosing between a good and a bad solution. But if we feel like something's better (the naming in this case), then I think we shouldn't be stuck with it because we think it might be confusing to users (especially because I think it will be an easy change).

Reply:
Hmm, actually, the more I think about it, the more I think `subject` is the better choice. I didn't care that much at first, because I thought it was an implementation detail (the friction I was thinking of was our own developer experience). But now I realized that this `name` is something the connector developer will see and use, while we will eventually expose the same field as `name` through the the schema registry API. It would be a bit confusing and inconsistent if we call the same field `name` in the connector SDK and `subject` in the schema registry API. Anyone with prior experience with the Confluent schema registry would also have an easier time if we used the same terminology across the board. It's not that it would be hard to get used to our terminology, but why create the barrier in the first place?

Reply:
Personally, I don't think it's that much of a barrier, especially because there's no permanent context switching (from developing a connector to working with the schema registry directly). It's also pretty infrequently used.

I also think that that providing a slightly smoother switch to Conduit is less important that us having a more meaningful name long term.

But this way or the other, I don't think either provides so much difference that we should spend too much time on it, so AFAIC we can just migrate to `subject`. :+1:

Reply:
Thank you for indulging me 😄

Reply:
lol :D I gotta return the favor of great reviews! :muscle: